dogfood/Dockerfile

```diff
@@ -312,8 +312,24 @@ COPY --from=go /tmp/bin /usr/local/bin
 COPY --from=rust-utils /tmp/bin /usr/local/bin
 COPY --from=proto /tmp/bin /usr/local/bin
 
+# Configure Nix without sandboxing - https://github.com/NixOS/nix/issues/2636#issuecomment-455302745
+RUN addgroup --system nixbld \
+  && adduser coder nixbld \
+  && for i in $(seq 1 30); do useradd -ms /bin/bash nixbld$i &&  adduser nixbld$i nixbld; done \
```

Comment:
What's the purpose of all of these users and groups? I can't find any reference to them in the linked issue. I don't see them added when doing a `curl -L https://nixos.org/nix/install | sh` either.

Reply:
https://nixos.org/manual/nix/stable/installation/multi-user.html#setting-up-the-build-users